bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/net/NetUtilTest.java

```diff
@@ -29,45 +30,79 @@
     public void testNetwork() {
         String network = NetUtil.getIpv4NetAddress("192.168.0.1", (short) 24);
         assertThat(network, is("192.168.0.0"));
+        String broadcast = NetUtil.getIpv4NetBroadcastAddress("192.168.0.1", (short) 24);
+        assertThat(broadcast, is("192.168.0.255"));
 
         network = NetUtil.getIpv4NetAddress("192.168.23.5", (short) 24);
         assertThat(network, is("192.168.23.0"));
+        broadcast = NetUtil.getIpv4NetBroadcastAddress("192.168.23.5", (short) 24);
+        assertThat(broadcast, is(" 192.168.23.255"));
```

Comment:
There is a space before the address, which is why the test case fails in travis (and locally) ;-)

Reply:
Indeed, thank you for that. I struggle getting the tests to run locally, I get: Required plug-in 'org.eclipse.jdt.junit4.runtime' could not be found.

Reply:
You cannot run the test as a plain JUnit test. You have to right click on https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core.test/CoreTests.launch

and run it as a `CoreTests` which will then run all core tests (including this one) as a J-Unit plug-in test. If that still fails you can edit this run configuration, click on the plugin tab and select the missing plugins there.